package.json

```diff
@@ -21,18 +21,26 @@
   "bugs": {
     "url": "https://github.com/codealchemist/youtube-audio-server/issues"
   },
+  "files": ["!/temp-audio/**", "src/*", "!/src/test"],
   "homepage": "https://github.com/codealchemist/youtube-audio-server#readme",
   "dependencies": {
     "express": "^4.17.1",
     "express-no-favicons": "0.0.1",
+    "ffmetadata": "^1.6.0",
     "fluent-ffmpeg": "^2.1.2",
     "minimist": "^1.2.0",
     "mkdirp": "^0.5.1",
+    "qs": "^6.9.1",
+    "retriable-download": "^2.1.0",
+    "sanitize-filename": "^1.6.3",
     "through2": "^2.0.3",
+    "updated-youtube-info": "^1.4.7",
+    "youtube-info": "^1.3.2",
```

Comment:
This one is currently deprecated and I it's not being used.
I think we need to remove it.

Reply:
Oh, and I just find out we should be able to use `ytdl-core` to get video info:
https://www.npmjs.com/package/ytdl-core#async-ytdlgetinfourl-options